assets/keymaps/default-linux.json

```diff
@@ -491,7 +491,8 @@
         "ctrl-alt-shift-c": "project_panel::CopyRelativePath",
         "f2": "project_panel::Rename",
         "enter": "project_panel::Rename",
-        "backspace": "project_panel::Delete",
+        "delete": "project_panel::Delete",
+        "ctrl-backspace": "project_panel::Delete",
```

Comment:
Is this really a thing in Linux? 
I know that ctrl-backspace removes words in that world, but usually Linux users have the `delete` key and do not need anything else, so let's remove it and wait for Linux users to add it if they need it.

Reply:
I am not sure that on Linux, so let remove it.

Reply:
I have reverted that file.